frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -620,8 +621,12 @@ mod tests {
 			assert_eq!(
 				multi_phase_events(),
 				vec![
-					Event::SignedPhaseStarted { round: 1 },
-					Event::SolutionStored { compute: ElectionCompute::Signed, prev_ejected: false }
+					Event::PhaseTransition { from: Phase::Off, to: Phase::Signed, round: 1 },
```

Comment:
There's a test in `lib.rs` that's only devoted to phase transition. Please add events check to that as well.

Reply:
already is